kornia/utils/helpers.py

```diff
@@ -316,7 +316,7 @@ def is_autocast_enabled(both: bool = True) -> bool:
         return False
 
     if both:
-        return torch.is_autocast_enabled() or torch.is_autocast_cpu_enabled()
+        return torch.is_autocast_enabled() or torch.is_autocast_enabled("cpu")
```

Comment:
maybe check which version this is introduced, and add an if checking the torch version for each case